tests/router/common.py

```
@@ -578,13 +578,40 @@ def _test_python_router_bindings(
+    kv_router_error = None
+
+    def _create_router():
+        nonlocal kv_router, kv_router_error
+        try:
+            with min_initial_workers_env(num_workers):
+                kv_router = KvRouter(
+                    endpoint=endpoint,
+                    block_size=block_size,
+                    kv_router_config=kv_router_config,
+                )
+        except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)